code/modules/trade/datums/_trade_station.dm

```diff
@@ -37,7 +39,7 @@
 
 /datum/trade_station/proc/init_src()
 	if(name)
-		crash_with("Some retard gived trade station a name before init_src, not thought name_pool. ([type])")
+		crash_with("Some retard gived trade station a name before init_src, overriding name_pool. ([type])")
```

Comment:
> "Someone gave a trade station a name before init_src, overriding name_pool! ([type])"

While this may seem absolutely banal, GitHub polices keywords, and a bad actor could use the keyword here to repo ban us like what happened to /tg/ station. Since we have no clue how many or what "threshold" of keywords constitutes GitHub banning a repo, I'd say it's best to remove them wherever we see them.